client/src/components/AuthForm/LogInForm/LogInForm.js

```diff
@@ -1,22 +1,31 @@
 import React, { useState } from 'react';
-import { connect } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import { Redirect } from 'react-router-dom';
-import PropTypes from 'prop-types';
+import validator from 'validator';
 import AuthHeader from 'components/AuthForm/AuthHeader/AuthHeader';
 import InputField from 'components/InputField/InputField';
 import Button from 'components/Button/Button';
-import ErrorMsg from 'components/Alert/Alert';
+import Alert from 'components/Alert/Alert';
 import AuthSwitch from 'components/AuthForm/AuthSwitch/AuthSwitch';
-import { login as loginAction } from 'actions/auth';
+import { login } from 'actions/auth';
+import { setAlert } from 'actions/alert';
 import styles from './LogInForm.module.scss';
 
-const LogInForm = ({ login, isAuthenticated, loading }) => {
+const LogInForm = () => {
   const [logInForm, setLoginForm] = useState({
     email: '',
     password: ''
   });
   const [hasAccount] = useState(false);
   const [showPass, setShowPass] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const {
+    auth: { isAuthenticated, loading },
+    alert
+  } = useSelector((state) => state);
+
+  const dispatch = useDispatch();
 
   const checkDisabled = () => {
     const result = Object.values(logInForm).filter((value) => value !== '');
```

Comment:
I think there is a better way to write this. 
The problem with Array.filter is that it iterates over the entire array, and it isn't required for this operation.
I recommend using something like Array.some, 

Object.values(logInForm).some(value => !!value);

That way as soon as the method finds a falsy value it will return, it won't iterate over the entire array.